scripts/docker-worker-linux/60-install-docker-worker.sh

```diff
@@ -54,21 +54,11 @@ worker:
     configPath: "${docker_worker_config}"
 # becomes part of docker-worker config
 workerConfig:
-    # loopbacks should work
-    deviceManagement:
-        loopbackAudio:
-            enabled: true
-        loopbackVideo:
-            enabled: true
     dockerWorkerPrivateKey: "${taskcluster_secrets_dir}/worker_cot_key"
     ed25519SigningKeyLocation: "${taskcluster_secrets_dir}/worker_ed25519_cot_key"
     ssl:
         certificate: "${taskcluster_secrets_dir}/worker_livelog_tls_cert"
         key: "${taskcluster_secrets_dir}/worker_livelog_tls_key"
-    shutdown:
-        enabled: false
-        afterIdleSeconds: 0
-cacheOverRestarts: "${worker_runner_state}"
```

Comment:
This line (`cacheOverRestarts`) should not be removed, it's worker-runner config

Reply:
but we only want it to be set if we want to cache config which we don't in these workers. should we set it to false instead or something?

Reply:
Why don't we want to cache config? If config isn't cached, restarting tc-worker-runner doesn't work because it tries to look for creds in userdata, and those creds have been expired / deleted.

Reply:
we don't restart worker-runner on these instances because they're intended to start once, get creds, run, then terminate.

If they can reuse creds it is actually a negative in this case.